qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -499,10 +499,16 @@ def test_mount_after_evicted_client(self):
 
         # create a new instance of mount_a's class with most of the
         # same settings, but mounted on mount_b's mountpoint.
-        m = cl(self.mount_a.ctx, self.mount_a.test_dir, self.mount_a.client_id,
-               self.mount_a.client_remote, self.mount_a.client_keyring_path,
-               self.mount_b.hostfs_mntpt, self.mount_a.cephfs_name,
-               self.mount_a.cephfs_mntpt, self.mount_a.ceph_brx_net)
+        m = cl(ctx=self.mount_a.ctx,
+               client_config=self.mount_a.client_config,
```

Comment:
This fails with kclient runs.

`kernel_mount.py::KernelMount` has the following ctor:

```
    def __init__(self, ctx, test_dir, client_id, client_remote,
                 client_keyring_path=None, hostfs_mntpt=None,
                 cephfs_name=None, cephfs_mntpt=None, brxnet=None, config={}):
```

The parameter is `config`. Moreover, the `config` param is not saved in the instance object, which means using `self.mount_a.client_config` is incorrect too.

Reply:
Fixed it.